Meta/CMake/pnp_ids.cmake

```diff
@@ -5,7 +5,7 @@ set(PNP_IDS_URL http://www.uefi.org/uefi-pnp-export)
 set(PNP_IDS_EXPORT_PATH ${CMAKE_BINARY_DIR}/pnp_ids.html)
 set(PNP_IDS_INSTALL_PATH ${CMAKE_INSTALL_DATAROOTDIR}/${PNP_IDS_FILE})
 
```

Comment:
is your close brace key broken too? :^)
(missing close brace in the commit title)

Reply:
Oops! Fixed